assets/js/index.js

```diff
@@ -1,14 +1,18 @@
-server = "https://api.losaltoshacks.com";
+// server = "https://api.losaltoshacks.com";
+server = "http://localhost:5000"
```

Comment:
Should point to prod

Reply:
We should maybe have this in a gitignored file

Reply:
I don't think it's that significant so gitignore file may not be necessary